src/controllers/update/remove.is.entity.registered.owner.controller.ts

```diff
@@ -1,15 +1,20 @@
 import { NextFunction, Request, Response } from "express";
 import * as config from "../../config";
 import { logger } from "../../utils/logger";
+import { ApplicationData } from "../../model";
+import { HasDisposedAllLandKey } from "../../model/data.types.model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
 
 export const get = (req: Request, res: Response, next: NextFunction) => {
   try {
     logger.debugRequest(req, `GET ${config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
 
     return res.render(config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE, {
       journey: config.JourneyType.remove,
       backLinkUrl: `${config.REMOVE_SOLD_ALL_LAND_FILTER_URL}${config.JOURNEY_REMOVE_QUERY_PARAM}`,
-      templateName: config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE
+      templateName: config.REMOVE_IS_ENTITY_REGISTERED_OWNER_PAGE,
+      [HasDisposedAllLandKey]: appData?.[HasDisposedAllLandKey]
```

Comment:
the sold-land-filter page asks 'Has the overseas entity disposed of all its property or land in the UK?' so the sold-land-filter controller and HasSoldLandKey will handle the 'disposed all land' value.  This controller is for 'Is the overseas entity currently listed on any land registry records as the registered owner of property or land in the UK?' so we'll need the key and model field to represent that. something like is_listed_as_land_or_property_owner

Reply:
A good point, but I would advise against any name changes, as soon as you attempt to change the name the code reacts badly, you enter a yes or a no , the error is displayed as though no input has been entered at and the user will not be able to go any further. Let's say I change all instances "owner_disposed" to "listed_as_property_owner" such that they correspond as before this is the result: 
![Screenshot 2023-12-07 at 08 43 37 (2)](https://github.com/companieshouse/overseas-entities-web/assets/38655987/1f64d946-34c1-4ffd-8778-bde5bcae976a)
There seems to be a very strong bond with the existing name.

Reply:
Updated the name but to something a bit shorter than the suggestion above.